diff --git a/node_modules/slick-carousel/slick/slick.js b/node_modules/slick-carousel/slick/slick.js
index 7aca613..886283f 100644
--- a/node_modules/slick-carousel/slick/slick.js
+++ b/node_modules/slick-carousel/slick/slick.js
@@ -2287,7 +2287,7 @@
     };
 
 
-    Slick.prototype.setSlideClasses = function(index) {
+    Slick.prototype.setSlideClasses = function(index, oldSlide) {
 
         var _ = this,
             centerOffset, allSlides, indexOffset, remainder;
@@ -2301,6 +2301,20 @@
             .eq(index)
             .addClass('slick-current');
 
+        if(oldSlide!=undefined){
+          if(index==0 && oldSlide!=1){
+              var _current = this.$slides.eq(this.$slides.length-1);
+              var __index = allSlides.index(_current);
+              var ss = allSlides.eq(__index+1);
+              ss.addClass('slick-current');
+          }
+          if(index==this.$slides.length-1 && oldSlide!=this.$slides.length-2){
+              var _current = this.$slides.eq(0);
+              var __index = allSlides.index(_current);
+              var ss = allSlides.eq(__index-1);
+              ss.addClass('slick-current');
+          }
+        }
         if (_.options.centerMode === true) {
 
             var evenCoef = _.options.slidesToShow % 2 === 0 ? 1 : 0;
@@ -2546,8 +2560,7 @@
 
         oldSlide = _.currentSlide;
         _.currentSlide = animSlide;
-
-        _.setSlideClasses(_.currentSlide);
+        _.setSlideClasses(_.currentSlide,oldSlide);
 
         if ( _.options.asNavFor ) {
 
